pipelines/cemba/cemba_methylcseq/README.md

```diff
@@ -1,40 +1,184 @@
-  <li>Split pool for PBS submission (CEMBA_split_pool.pl)
-  <li>Bismark wrapper (CEMBA_bismark.pl, bismark v0.14.4)
-  <li>Remove clonal reads (picard tools 1.141)
-  <li>Reads filter -- for mouse data, just filter on MAPQ (samtools 1.3).
-  <li>Gzip demultiplexed fastq files (CEMBA_gzip.pl)
-  <li>Move fastq files for failed cells to recycle (CEMBA_fastq_filter.pl)
-  <li>Generate mapping summary (CEMBA_summary_dualSE_mapping.pl)
-  <li>Merge mapping summary (CEMBA_summary_dualSE_mapping_merge.pl)
-  <li>Methylpy mpileup and allc tables
-  <li>Create allc file (allc_SAMPLE.tsv.bgz) and corresponding tabix index (.tbi)
-  <li>Create binc files, genebody, promoter (Will be available in next methylpy release)
-</ol>
-
-## Submitting Workflows to CaaS
-
-To run workflows in the cloud, first generate an options file containing CaaS secrets.
-*DO NOT* check the generated file in.
-```bash
-$ ./scripts/inject-caas-secrets <path/to/options.json> > <path/not/in/repo/options.caas.json>
-```
-
-Then submit your workflow with the generated options in one of two ways:
-
-### Manually via Swagger
-
-Follow the instructions [here](https://docs.google.com/document/d/1od3J2IgO5z27BKh8DEP4lbPilDE1t6f4xk8DdvkxGmI/edit#).
-When told to upload options JSON, choose to upload the options file generated by `inject-caas-secrets`.
-
-### Via the CLI
-
-TODO after the [`cromwell-tools` refactor](https://github.com/broadinstitute/cromwell-tools/pull/43) exposes CaaS-related
-options on the CLI.
+| Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| [CEMBA_v1.0](CEMBA.wdl) | July 28, 2020 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in warp or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
+
+# Table of Contents
+
+- [CEMBA Pipeline Overview](#cemba-pipeline-overview)
+  * [Introduction to the CEMBA Workflow](#introduction-to-the-cemba-workflow)
+  * [Quick Start Table](#quick-start-table)
+- [Set-up](#set-up)
+  * [CEMBA Installation and Requirements](#cemba-installation-and-requirements)
+  * [Inputs](#inputs)
+    + [Sample data input](#sample-data-input)
+    + [Additional Inputs](#additional-inputs)
+- [CEMBA Tasks and Tools](#cemba-tasks-and-tools)
+  * [CEMBA Task Summary](#cemba-task-summary)
+- [Outputs](#outputs)
+- [Versioning](#versioning)
+- [Have Suggestions?](#have-suggestions)
+
+
+
+# CEMBA Pipeline Overview
+
+<img src="CEMBA.png" width="750">
+
+## Introduction to the CEMBA Workflow
+
+CEMBA is a pipeline developed by the Data Coordination Platform (DCP) of the [Human Cell Atlas (HCA) Project](https://data.humancellatlas.org/) that supports processing of multiplexed single-nuclei bisulfite sequencing data. It is an alignment and methylated base calling pipeline that trims adaptors, attaches cell barcodes, aligns reads to the genome, filters reads based on quality and creates a VCF with methylation-site coverage. 
```

Comment:
This pipeline was actually developed in collaboration with the BRAIN initiative, we don't use it for HCA.

Reply:
I updated this to BRAIN Initiative.